### PR TITLE
[stable/prometheus-operator] fixes and addition in README.md

### PR DESCRIPTION
**Pull request by aliartiza75**
- Documentation regarded some parameter are modified and updated.

### CODE REVIEW

**Comment by aliartiza75** [comment]
@vsliouniaev I have just updated the document why checks have failed?

File: stable/prometheus-operator/README.md [hunk 1]
```diff
@@ -307,6 +307,8 @@ The following tables list the configurable parameters of the prometheus-operator
 | `grafana.sidecar.datasources.defaultDatasourceEnabled` | Enable Grafana `Prometheus` default datasource` | `true` |
 | `grafana.sidecar.datasources.label` | If the sidecar is enabled, configmaps with this label will be loaded into Grafana as datasources configurations | `grafana_datasource` |
 | `grafana.rbac.pspUseAppArmor` | Enforce AppArmor in created PodSecurityPolicy (requires rbac.pspEnabled) | `true` |
+| `grafana.rbac.create` | Create rbac for grafana | `true` |
```
Comment (vsliouniaev) [inline stable/prometheus-operator/README.md:310]: Generally chats don't document sub-chart variables. If you document one, should you document them all?

**Review by vsliouniaev — CHANGES_REQUESTED** [review]

**Comment by vsliouniaev** [comment]
You need to sign off commits with `git commit - s` and also bump the chart version. Clicking into the checks will contain logs about what the issues are.

File: stable/prometheus-operator/README.md (hunk 1 above)
Comment (aliartiza75) [reply stable/prometheus-operator/README.md:310]: @vsliouniaev are you suggesting me to document all of them?

Comment (vsliouniaev) [reply stable/prometheus-operator/README.md:310]: I'm suggesting that you omit documenting them

Comment (aliartiza75) [reply stable/prometheus-operator/README.md:310]: @vsliouniaev ack

**Pull request closed without merging**